src/jobs/models/manager.ts

```diff
@@ -53,11 +53,11 @@ export class JobManager {
       if (stagesReq !== undefined && stagesReq.length > 0) {
         const stages: StageCreateModel[] = stagesReq;
         stagesInput = stages.map((stage) => {
-          const { type, ...rest } = stage;
+          const { type, isWaiting, ...rest } = stage;
           const stageFull = Object.assign(rest, {
-            xstate: persistenceSnapshot,
+            xstate: getInitialXstate(stage),
             name: type,
-            status: StageOperationStatus.CREATED,
+            status: (isWaiting ?? false) ? StageOperationStatus.WAITING : StageOperationStatus.CREATED,
```

Comment:
```suggestion
            status: (isWaiting === true) ? StageOperationStatus.WAITING : StageOperationStatus.CREATED,
```
i didnt check it actually works so test it.